composer.json

```
@@ -3,47 +3,73 @@
   "description": "Composer project template for the Apigee Developer Portal Drupal distribution.",
   "type": "project",
   "license": "GPL-2.0-or-later",
+  "homepage": "https://www.drupal.org/project/apigee_devportal_kickstart",
+  "support": {
+    "docs": "https://www.drupal.org/docs/8/modules/apigee-developer-portal-kickstart"
+  },
   "repositories": [
     {
       "type": "composer",
       "url": "https://packages.drupal.org/8"
     }
   ],
-  "config": {
-    "process-timeout": 0
-  },
   "require": {
     "composer/installers": "^1.2",
-    "cweagans/composer-patches": "^1.6.5",
-    "drupal-composer/drupal-scaffold": "^2.5",
+    "drupal/core-composer-scaffold": "^8.8",
+    "drupal/core-project-message": "^8.8",
+    "drupal/core-recommended": "^8.8",
     "apigee/apigee_devportal_kickstart": "^1.0.0",
     "drupal/apigee_m10n": "^1.0.0",
     "drupal/commerce": "^2.13",
```

The latest commerce requires commerce >=8.x-2.16, so we can bump this up.
```suggestion
    "drupal/commerce": "^2.16",
```